src/components/Dropdown/Dropdown.tsx

```diff
@@ -21,7 +21,13 @@ export interface DropdownProps {
   initialValue?: string,
   parentQuery?: string,
   onSelect?: (value: string, index: number, focusedItemData: Record<string, unknown> | undefined) => void,
-  onToggle?: (isActive: boolean, value: string) => void,
+  onToggle?: (
+    prevValue: string,
+    value: string,
+    index: number,
+    focusedItemData: Record<string, unknown> | undefined,
+    isActive: boolean
```

Comment:
Since we moved `isActive` last, are there any usages of `Dropdown` that supply `onToggle` that need updating?